session_analyzer.py

```diff
@@ -14,14 +14,16 @@ def __init__(self):
 
     @asyncio.coroutine
     def analyze(self):
-        session = None
-        session_key = yield
-        try:
-            session = self.r.get(session_key)
-            session = json.loads(session.decode('utf-8'))
-        except (redis.ConnectionError, TypeError) as e:
-            pass
-        stats = self.create_stats(session)
+        while True:
```

Comment:
I would add a sleep in the loop to avoid using tons of CPU if something goes wrong.